File: app/src/main/java/ws/temp/jadestern/activity/PrefaceActivity.java
package ws.temp.jadestern.activity;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import ws.temp.jadestern.R;
import ws.temp.jadestern.model.AccountModel;

public class PrefaceActivity extends BaseActivity {
    private static final String TAG = PrefaceActivity.class.getSimpleName();

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // TODO: 16/02/27 add splash
        setContentView(R.layout.activity_preface);
        AccountModel accountModel = new AccountModel();

        if (accountModel.getAccountLength() < 1)
            // start add user activity
            AccountModel.startAddAccountActivity(this);
        else
            startActivity(new Intent(this, MainActivity.class));

        finish();
    }
}

File: app/src/main/java/ws/temp/jadestern/activity/PrefaceActivity.kt
package ws.temp.jadestern.activity

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import ws.temp.jadestern.R

class PrefaceActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_preface)
    }

    companion object {
        private val TAG = PrefaceActivity::class.java.simpleName
    }
}
